numpyro/distributions/continuous.py

```diff
@@ -668,6 +669,68 @@ def cdf(self, x):
         return 1 - self.base_dist.cdf(1 / x)
 
 
+class Gompertz(Distribution):
+    """Gompertz Distribution sampling utilities.
```

Comment:
Use raw string to avoid `\\` in math mode.

Reply:
done